### DISCUSSION TITLE
Rack height more than 100 RU is not supported?

### DISCUSSION THREAD

**Discussion by redstarsachin** [netbox-community/netbox#17031] [General]
In Netbox documentation it says: 

Rack height is measured in rack units (U); racks are commonly between 42U and 48U tall, but NetBox allows you to define racks of arbitrary height.

But when I try to create Rack with more than 100 RU I couldn't. 

error: Ensure this value is less than or equal to 100. 

![image](https://github.com/user-attachments/assets/c943af25-cbae-4388-aa56-8bb9913a015f)

**Comment by jsenecal** [comment]
Why do you need more than the tallest 55RU rack exactly ? A 100RU rack would be 175 inches/~14.6feet/4.445 Meters tall, which doesn't make any sense.

**Reply by redstarsachin** [reply]
I want to create an IDF rack which has more than 100 patch panels.

**Reply by ZPrimed** [reply]
Given that the intent of the software is to model the real world, I think the question here is do you actually have a 14.6 foot tall rack that has 100+ U in it?

If your patch panels are in different racks, is there some reason you can't just define each of those racks?

**Reply by candlerb** [reply]
> Rack height is measured in rack units (U); racks are commonly between 42U and 48U tall, but NetBox allows you to define racks of arbitrary height.

I think that is a documentation error; it should say NetBox allows you to define racks up to 100U.

> I want to create an IDF rack which has more than 100 patch panels.

Can you share a link to the panel and/or rack design?

If these are very thin (0.5U) panels then Netbox can model them directly.  If these are panels which sit side-by-side in the same rack unit, then you can have a 1U Device which has either Modules or Child Devices to hold the panels.

For example, [these](https://www.completeconnect.co.uk/mx-xpress-pre-terminated-modular-fibre/) come as a 1U frame that takes 5 cassettes, or a 2U frame that takes 12 cassettes, and can be modelled as such in Netbox.

**Reply by redstarsachin** [reply]
![image](https://github.com/user-attachments/assets/d06963d6-cdcd-4fb5-b91b-ba3420c781ca)

**Reply by candlerb** [reply]
Can you share a link which contains images and/or specifications of the *physical* ODFs or panels, and how they are laid out?

Your table shows `{A..M}{1..5}` which is 65 rows, then 65 lots of "cable management", then another 65 `{N..Z}{1..5}`. But that doesn't show how these physically relate to positions in a 19" rack.

1U is, [by definition](https://en.wikipedia.org/wiki/Rack_unit), 1.75 inches. I don't think it's likely that you have 195 x 1.75" panels stacked vertically above each other.

**Reply by redstarsachin** [reply]
The ODF I have is something like this, panels on both sides.

<img width="251" alt="image" src="https://github.com/user-attachments/assets/b600c18a-e9e5-42f4-adcf-4286b256b91e">

**Reply by jsenecal** [reply]
These are technically two different Racks, or h-frames, bolted together with a fiber management accessory in between them. You should treat them individually in Netbox.
